web/app/src/components/EndpointGroup.vue

```diff
@@ -7,11 +7,12 @@
             {{ collapsed ? '&#9660;' : '&#9650;' }}
           </span>
           {{ name }}
-          <span v-if="healthy" class="float-right text-green-600 w-7 hover:scale-110" title="Operational">
-            <CheckCircleIcon />
+          <span v-if="unhealthyCount" class="float-right text-yellow-500 text-sm w-7 hover:scale-110" title="Partial Outage">
+            <span v-if="unhealthyCount<100" class="rounded-xl bg-yellow-500 text-gray-900 font-bold leading-6 float-right w-6 h-6 text-center hover:scale-110">{{unhealthyCount}}</span>
```

Comment:
Why 100? Seems like an arbitrary number. Always displaying the number if it's non-0 seems like a better option imo

Reply:
I chose 100 because it does not break design. More digits won't fit in the circle. I'll experiment with alternatives

Reply:
updated, please review

Reply:
Awesome, looks great!